components/proxy/src/main/java/com/hotels/styx/startup/StyxServerComponents.java

```diff
@@ -262,6 +274,11 @@ public Builder loggingSetUp(LoggingSetUp loggingSetUp) {
             return this;
         }
 
+        public Builder httpMessageFormatter(HttpMessageFormatter httpMessageFormatter) {
+            this.httpMessageFormatter = httpMessageFormatter;
+            return this;
+        }
```

Comment:
This presumably is unused and needs to be removed?

Reply:
removed